cypress/e2e/listingDetails/completedLotteryListing.e2e.ts

```diff
@@ -0,0 +1,96 @@
+const listings = {
+  COMPLETED_LOTTERY: {
+    id: "a0W8H000000AmpKUAS",
+  },
+}
+
+const MOBILE_VIEWPORT_HEIGHT = 680
+const MOBILE_VIEWPORT_WIDTH = 420
+
+const visitListing = (mobile, language) => {
+  const langPart = language ? `/${language}` : ""
+  if (mobile) {
+    cy.viewport(MOBILE_VIEWPORT_WIDTH, MOBILE_VIEWPORT_HEIGHT)
+  }
+  cy.visit(`${langPart}/listings/${listings.COMPLETED_LOTTERY.id}?react=true`)
+}
+
+const clickLotteryResultsButton = (mobile) => {
+  if (mobile) {
+    cy.get(".detail-header__title").contains("Lottery").click()
+  }
+  cy.get("button").contains("View Lottery Results").click()
+}
+
+const searchForLotteryResults = () => {
+  cy.get("#lotterySearchNumber").type("01100972")
+  cy.get("form button svg").click()
```

Comment:
hmm, this one feels weird. is `svg` necessary? have we considered [submit](https://docs.cypress.io/api/commands/submit#Syntax)?